packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/textbox.tests.js

```diff
@@ -220,6 +220,20 @@ QUnit.module('options changing', {
         }
     });
 
+    QUnit.test('The validation error should be null after removing the mask (T1214604)', function(assert) {
```

Comment:
The fix is at the texteditor masks level => the test should be also at the texteditor masks label. Please move your test to `textEditorsParts/masks.tests.js` file

Reply:
[c0783df5eff72f2e9d00601f6521ea5721a42bf0]